### PR TITLE
fix: add timestamp conversion mock

### PR DESCRIPTION
**Pull request by sanpj2292**
## Description of the change

Changes only applicable for GAOC tests

- move timestamp conversion logic to helper & use the helper object
- include common mock logic
- add mocks for all relevant test-cases
- clean-up unnecessary imports

Resolves INT-886

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/google_adwords_offline_conversions/helper.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const moment = require('moment');
```
Comment (yashasvibajpai) [inline src/v0/destinations/google_adwords_offline_conversions/helper.js:1]: Generic question:
Can we have this helper in some general util directory or is it very specific to GAOC?

Comment (sanpj2292) [reply src/v0/destinations/google_adwords_offline_conversions/helper.js:1]: Good point 👍🏼 
I think this logic is specific to `GAOC` alone. Not sure if other destinations also require same logic, hence not preferring to move it to a common place for now.

**Comment by sanpj2292** [comment]
Resolves INT-886

**Comment by sanpj2292** [comment]
Used `moment-timezone` instead of `moment` to make it easy for us to write tests.

My Observations:
- `moment-timezone` import `moment` & returns `moment` in `tz` object. So `moment` functionality is not lost.
- `moment-timezone` provides us with capability of choosing timezones at will which makes it a lot powerful

**Review by koladilip — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Review by yashasvibajpai — APPROVED** [review]

**Pull request merged**